.github/workflows/ci.yml

```diff
@@ -36,3 +40,5 @@ jobs:
 
     - name: Run tests
       run: npm test
+      env:
```

Comment:
I think edge is universally available everywhere as well as chrome.

Reply:
I don't have it on my linux machine. But if we are okay with some of the projects failing on some of the machines, I can remove all envs.

Reply:
I don't have it on my linux machine. But if we are okay with some of the projects failing on some of the machines, I can remove all envs.